components/Banner/tests/snapshot/MDCBannerSnapshotTests.m

```diff
@@ -19,9 +19,12 @@
 #import "MaterialTypographyScheme.h"
 
 static NSString *const kBannerShortText = @"tristique senectus et";
+static NSString *const kBannerMiddleLengthText =
```

Comment:
Tip: What makes these "middle" versus "long"? It looks like the difference is one word?

Reply:
The different was one word. The bug happens on an edge case, and I didn't want to change to long and extraLong for diff concerns.